Makefile

```diff
@@ -179,33 +178,30 @@ vendor: $(GVT)
 	@echo restoring vendor
 	$(GOPATH)/bin/gvt restore
 
-$(GOPATH)/bin/bats:
-	@./tests/vendor/github.com/sstephenson/bats/install.sh $(GOPATH)
-
-integration-tests: $(GOPATH)/bin/bats $(GOVC) components isos vic-machine
+integration-tests: $(GOVC) components isos vic-machine
 	@echo Running integration tests
-	@GOVC=$(GOVC) $(GOPATH)/bin/bats -t tests
+	@GOVC=$(GOVC) ./tests/vendor/github.com/sstephenson/bats/libexec/bats -t tests
 
 TEST_DIRS=github.com/vmware/vic/cmd/tether
 TEST_DIRS+=github.com/vmware/vic/cmd/imagec
 TEST_DIRS+=github.com/vmware/vic/cmd/vicadmin
 TEST_DIRS+=github.com/vmware/vic/cmd/rpctool
 TEST_DIRS+=github.com/vmware/vic/cmd/vic-machine
-TEST_DIRS+=github.com/vmware/vic/portlayer
+TEST_DIRS+=github.com/vmware/vic/lib/apiservers/portlayer
+TEST_DIRS+=github.com/vmware/vic/lib/install
+TEST_DIRS+=github.com/vmware/vic/lib/portlayer
 TEST_DIRS+=github.com/vmware/vic/pkg
-TEST_DIRS+=github.com/vmware/vic/apiservers/portlayer
-TEST_DIRS+=github.com/vmware/vic/install
 
 
 test:
 	@echo Running unit tests
 	# test everything but vendor
 ifdef DRONE
 	@echo Generating coverage data
-	@scripts/coverage.sh $(TEST_DIRS)
+	@infra/scripts/coverage.sh $(TEST_DIRS)
```

Comment:
/tests is for the integration tests whereas coverage.sh is used to process the `go test` output.